### ISSUE TITLE
J+H ungrabs the ruler

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab#190]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/455
When using keyboard nav, jumping the ruler to home position ungrabs the ruler.
**Steps to reproduce**
1. Use keybard nav and go to the ruler
2. Grab the ruler
3. Move it somewhere else
4. Press J and H

**Visuals**
![jhungrabs](https://user-images.githubusercontent.com/41024075/68433269-578dd080-0173-11ea-9ade-860c40dd5338.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.16/phet/gravity-force-lab_all_phet.html
Version: 2.2.0-dev.16 2019-11-06 21:45:42 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.87 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
This is expected behavior, bringing the ruler home "let's go" of it. Closing
